src/Core/Math/ColorPresets.hpp

```diff
@@ -8,7 +8,7 @@ namespace Core {
 
 /// Colors are defined as vector4, i.e. 4 floats in RGBA order.
 /// displayable colors should have all their coordinates between 0 and 1.
-namespace Colors {
+namespace Math {
```

Comment:
Here we should have a container class Color.
It's odd to have Math::ToARGB32
but we should have Math::ColorUtils::ToARGB32
So a good alternative is to have all these function as static member function of a template class ColorUtils<C>

Reply:
:+1: